### DISCUSSION TITLE
v.1.8 - BT-514-Organization-Company

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#611] [Q&A] [answered]
Hi Labs, 

this fields is marked as mandatory (no coditions). But if it wasnt filled and sent for validation at CVS server. It response that everything is ok. Can you check validation rules for this field ?
![image](https://github.com/OP-TED/eForms-SDK/assets/11737559/ac159c07-9ea8-4098-b70d-e4c7b0b3ab32)

**Comment by YvesJo** [comment, marked as answer]
Hi,
The rule is a mandatory stage 2 rule with ND-Company being the context for it to get fired. This only applies to Company, not to touchpoint.
You could may be share the generated XML to the helpdesk for investigation.
KR,

**Reply by elerik** [reply]
Hi, 
that was my mistake. Thanks for reminding. After revisiting, validation failed which was expected while its mandatory field. 
Iam marking your answer.
